rocky/rocky/scheduler.py

```diff
@@ -196,30 +198,43 @@ def list_tasks(
         **kwargs,
     ) -> PaginatedTasksResponse:
         kwargs = {k: v for k, v in kwargs.items() if v is not None}  # filter Nones from kwargs
-        res = self._client.get("/tasks", params=kwargs)
+        endpoint = "/tasks"
+        logger.debug("Making request to %s with args %s", endpoint, kwargs)
```

Comment:
The debug statement seems unnecessary since requests are already logged by the HTTP client. If needed, we can make this logging configurable. This would enable consistent and fine-tuned logging to meet the user's needs

Reply:
Ah, I did not see the HTTP client logging during debugging, how do I make this visible within the KAT environment?

Reply:
Found it